lib/rules/no-duplicate-at-import-rules/index.mjs

```diff
@@ -36,7 +37,7 @@ const rule = (primary) => {
 
 			// extract uri from url() if exists
 			const uri =
-				firstParam.type === 'function' && firstParam.value === 'url' && firstParam.nodes[0]
+				isValueFunction(firstParam) && firstParam.value === 'url' && firstParam.nodes[0]
```

Comment:
TS doesn't narrow the type when using `isUrlFunction`.

Reply:
[suggest] How about changing the return type of `isUrlFunction` like this?

```diff
 /**
  * @param {import('postcss-value-parser').Node} node
- * @returns {boolean}
+ * @returns {node is import('postcss-value-parser').FunctionNode}
  */
 export default function isUrlFunction(node) {
 	return isValueFunction(node) && node.value.toLowerCase() === 'url';
```

Ref:
https://github.com/stylelint/stylelint/blob/ee3118d2460cddf469f959b5438d8aeab7e9584e/lib/utils/typeGuards.mjs#L60-L64

Reply:
The same change can be applied to `isVarFunction`, too.